docs/airflow/dags-maintenance.md

```diff
@@ -1,60 +1,59 @@
 (dags-maintenance)=
 # Production DAGs Maintenance
 
-_Last edited: May 5, 2022_
+We use [Airflow](https://airflow.apache.org/) to orchestrate our data ingest processes. This page describes how to handle cases where an Airflow [DAG task](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html) fails.
 
-## Re-running or clearing DAGs
+## Monitoring DAGs
 
-Sometimes DAGs will fail in production. Below are considerations to take into account when re-running or clearing DAGs to address failures.
+When an Airflow DAG task fails, it will alert to the `#data-infra-alerts` channel in Slack.
 
-There are roughly two categories of Airflow DAGs in our system.
+In that case, someone should respond according to the considerations described below and in each individual DAG's documentation (available in [each DAG subdirectory](https://github.com/cal-itp/data-infra/tree/main/airflow/dags) in the `data-infra` GitHub repository).
+
+### Considerations for re-running or clearing DAGs
+
+Below are considerations to take into account when re-running or clearing DAGs to address failures. You can [consult the individual DAG's documentation](https://github.com/cal-itp/data-infra/tree/main/airflow/dags) for information on which of the following categories each DAG falls into.
+
+#### Now vs. data interval processing
+
+There are roughly two types of Airflow DAGs in our system:
 * "Now" DAGs - mostly for executing code on a schedule (often scraping current data, or a fancy cron job), NOT orchestrating distributed processing of existing data
   * **When these DAGs fail, and you'd like to re-run them, you should execute a new manual run rather than clearing a historical run.**
   * Only the actual execution time matters if relevant (usually for timestamping data or artifacts)
   * Generally safe but not useful to execute multiple times simultaneously
-  * There is no concept of backfilling via these DAGs; they may output data/artifacts that require "backfilling" (i.e. modifying existing data) via a script or notebook
-* "Data interval processing" DAGs - orchestrates processing of previously-captured data, or data than can be retrieved in a timestamped manner (e.g. Amplitude)
+  * There is no concept of backfilling via these DAGs
+* "Data interval processing" DAGs - these DAGs orchestrate processing of previously-captured data, or data than can be retrieved in a timestamped manner (e.g. Amplitude)
```

Comment:
nit: could you say `GTFS Schedule` or something since we don't do this for amplitude anymore

Reply:
not saying GTFS schedule here because that's in the first category; just deleting the parenthetical

Reply:
oh I meant the parsing/validating but yeah seems fine

Reply:
ah yeah my interpretation was that the first part of the sentence was for "our" saved data and the second was for APIs or syncs that allow time range lookups